components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/CommentMappingUtil.java

```diff
@@ -48,6 +53,43 @@ public static CommentDTO fromCommentToDTO(Comment comment) throws APIManagementE
         return commentDTO;
     }
 
+    /**
+     * Converts a Comment object into corresponding REST API CommentDTO object with User Info
+     *
+     * @param comment comment object
+     * @return CommentDTO
+     */
+    public static CommentDTO fromCommentToDTOWithUserInfo(Comment comment, Map<String,
+            Map<String, String>> userClaimsMap)  throws APIManagementException {
+        CommentDTO commentDTO = fromCommentToDTO(comment);
+        if (userClaimsMap.get(comment.getUser()) != null) {
+            Map userClaims = userClaimsMap.get(comment.getUser());
+            CommenterInfoDTO commenterInfoDTO = new CommenterInfoDTO();
+            commenterInfoDTO.setFullName((String) userClaims.get(APIConstants.FULL_NAME));
+            commenterInfoDTO.setFirstName((String) userClaims.get(APIConstants.FIRST_NAME));
+            commenterInfoDTO.setLastName((String) userClaims.get(APIConstants.LAST_NAME));
+            commentDTO.setCommenterInfo(commenterInfoDTO);
+        }
+        return  commentDTO;
+    }
+
+    /**
+     * Retrieve userClaims from UserStore and save it in a cache map.
+     *
+     * @param username        commenter username
+     * @param userClaimsMap   cache map with user deatils
+     * @return Map<String, Map<String, String>>
+     */
+    public static  Map<String, Map<String, String>> retrieveUserClaims(String username,
+                                                                       Map<String, Map<String, String>> userClaimsMap)
+            throws APIManagementException {
```

Comment:
Format the code

Reply:
We can't include `Map<String, Map<String, String>> userClaimsMap` in line 83. It is exceeding the number of characters limit

Reply:
can we merge 85 and 84 to a single line?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9454/commits/dbe6f847730212f0db8456b3628c917d31a75961